descriptor/fingerprint/src/test/java/org/openscience/cdk/fingerprint/AtomPairs2DFingerprintTest.java

```diff
@@ -0,0 +1,87 @@
+/* This work is the product of a US Government employee as part of his/her regular duties
+ * and is thus in the public domain.
+ * 
+ * Author: Lyle D. Burgoon, Ph.D.
```

Comment:
It is a community standard to add your email address too. The reason for that is, if someone would even like to change the license (or resolve some copyright question), there is some guidance how to reach the author. There are legal obligations to do ones best to contact original authors, and this custom is to help out a bit.